ucsf_webedit_lite/ucsf_webedit_lite.features.inc

```diff
@@ -0,0 +1,15 @@
+<?php
+
+// This file is deprecated and can be removed.
+// Please remove include_once('ucsf_webedit_lite.features.inc') in ucsf_webedit_lite.module as well.
```

Comment:
Yes, because this will now be a submodule of ucsf_webedit. @stopfstedt   Should I leave this to you or let me push a commit to both repo's?